### PR TITLE
Dissociate number id chapter

### PR DESCRIPTION
**Pull request by emainguelag**
dissociate number from ID into chapter table

### CODE REVIEW

File: src/View/Chapter/_admin_form.html.twig [hunk 1]
```diff
@@ -4,6 +4,10 @@
 		<label for="name">Nom</label>
 		<input class="form-control" type="text" value="{{ chapter.name }}" placeholder="Entrez un nom court" name="name" id="name" required>
 	</p>
+	<p class="form-group">
+		<label for="number">Numéro du chapitre</label>
+		<input class="form-control" type="text" value="{{ chapter.number }}" placeholder="Entrez un numéro de chapitre dont vous aurez vérifié la cohérence" name="number" id="number" required>
```
Comment (aldaitzwild) [inline src/View/Chapter/_admin_form.html.twig]: Why can we put something else than a number in this input ?

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by emainguelag** [comment]
form control numeric added on chapter number

File: src/View/Chapter/_admin_form.html.twig (hunk 1 above)
Comment (aldaitzwild) [reply src/View/Chapter/_admin_form.html.twig]: The problem is still here the type should be "number"

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

**Comment by emainguelag** [comment]
type number corrected

**Review by aldaitzwild — APPROVED** [review]

**Pull request merged**